azure/bastion/outputs.tf

```diff
@@ -11,9 +11,4 @@ output "location" {
 output "name" {
   description = "Name of the bastion."
   value       = azurerm_bastion_host.main.name
-}
-
-output "resource_group_name" {
```

Comment:
might be worth still passing the resource group name on the output but taking it's value directly from the output of the bastion